documentation/modules/managing/proc-accessing-resource-status.adoc

```diff
@@ -18,10 +18,12 @@ This procedure describes how to find the status of a custom resource.
 +
 [source,shell,subs="+quotes,attributes"]
 ----
-kubectl get kafka _<kafka_resource_name>_ -o jsonpath='{.status}'
+kubectl get kafka <kafka_resource_name> -o jsonpath='{.status}' | jq
 ----
 +
 This expression returns all the status information for the specified custom resource. You can use dot notation, such as `status.listeners` or `status.observedGeneration`, to fine-tune the status information you wish to see.
++
+Using `jq` makes it easier to read the output.
```

Comment:
Do we define anywhere what `jq` is and link to it?

Reply:
We can define it in the description: "Using the `jq` command line JSON parser tool makes it easier to read the output."
Not sure we need a link when the tool is so easily found?

Reply:
I would probably add a link to GitHub for example (https://github.com/jqlang/jq I guess?). I do not think it is really clear.

Reply:
Okay. I've added a link where we mention the tool.